exercises/practice/twelve-days/.meta/example.cpp

```diff
@@ -0,0 +1,51 @@
+#include "twelve_days.h"
+#include <array>
+#include <string>
+
+namespace twelve_days {
+
+std::string verse(int day) {
+    const std::array<std::string, 13> ordinals = {
```

Comment:
This doesn't buy you anything (this still needs to be created each time `verse` is called). If you want to stick to using `std::string`, `std::vector` is the better container. But you could also just use my suggestion from the previous review.